AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/backgroundfilter/CwtEnum.h

```diff
@@ -0,0 +1,52 @@
+//
```

Comment:
does this file need to be in SDK layer?

Reply:
Unfortunately for now yes. The ML library is built statically and I haven't looked into the build problems to switch it to a dynamic one. This is needed to make the Objective-C reflection work and force it to link the symbols dynamically.